### ISSUE TITLE
Exception: `TypeError: decode() argument 1 must be string, not None`

### ISSUE THREAD

**Issue by paulmenzel** [masmu/pulseaudio-dlna#127]
Using pulseaudio-dlna built from master, everything works fine. On the command line I see an exception though, which does not have any noticeable effect though.

```
$ git describe --tags
0.4.6-66-gb26af4c
$ dpkg-buildpackage -b -us -uc
$ sudo dpkg -i ../pulseaudio-dlna_0.4.6_all.deb
$ pulseaudio-dlna
[…]
Exception happened during processing of request from ('192.168.1.27', 1900)
Traceback (most recent call last):
  File "/usr/lib/python2.7/SocketServer.py", line 599, in process_request_thread
    self.finish_request(request, client_address)
  File "/usr/lib/python2.7/SocketServer.py", line 334, in finish_request
    self.RequestHandlerClass(request, client_address, self)
  File "/usr/lib/python2.7/SocketServer.py", line 655, in __init__
    self.handle()
  File "/usr/lib/python2.7/dist-packages/pulseaudio_dlna/listener.py", line 38, in handle
    packet = self.request[0].decode(guess['encoding'])
TypeError: decode() argument 1 must be string, not None
----------------------------------------
```

**Comment by masmu** [comment]
Thanks for the bug report. I think I've added a fix for that. Could you checkout current master and try again?

**Comment by paulmenzel** [comment]
The issues is indeed fixed with pulseaudio-dlna commit f097bc5 (0.4.6-84-gf097bc5).
